README.md

```diff
@@ -1,24 +1,44 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Set up a "Hello world" project with react-rails gem
 
-Things you may want to cover:
+> A simple application for study purposes. It combines rails as a back-end and react as a front-end library.
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby on Rails
+- React
 
-* Configuration
+## Getting Started
+To get a local copy up and running follow these simple example steps.
 
-* Database creation
+- Copy and clone this link locally `https://github.com/tugsem/my_new_rails_app.git`.
+- Go to the repository folder in your command prompt `cd my_new_rails_app`.
 
-* Database initialization
+- Follow these steps to run the app:
+  - run `bundle` and `yarn install` to install all dependencies and ruby gems
+  - run `rails db:create`,`rails db:migrate`,`rails db:seed` to create database
+  - run `rails s`
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you also include the `npm run build` or `yarn build` command in your documentation before running the application(i.e. to come before `rails s`). Because running that command, the user will get this error: `The asset "application.js" is not present in the asset pipeline in Rails`.
You can update steps to run the app like this:
```
- Follow these steps to run the app:
  - run `bundle install` to  install all ruby gems
  - run `yarn install` or `npm install` to install all dependencies
  - run `rails db:create`,`rails db:migrate`,`rails db:seed` to create database
  - run `npm run build` or `yarn build` for webpacker handles javascript compilation in Rails.
  - run `rails s`
```
For more details about the above issue, I suggest you read this **[article](https://stackoverflow.com/questions/55912897/the-asset-application-js-is-not-present-in-the-asset-pipeline-in-rails-6)**.